### PR TITLE
GH1483: Removed the duplicate entries for xunit core

### PR DESCRIPTION
**Pull request by phillipsj**
Removed the duplicate entries and succesfully performed a build from
Visual Studio and using Cake.  All tests pass.

### CODE REVIEW

File: src/Cake.Testing.Xunit/project.json [hunk 1]
```diff
@@ -30,14 +30,10 @@
         ]
       },
       "imports": "dnxcore50",
-      "dependencies": {
-        "xunit.core": "2.2.0-beta2-build3300"
-      }
+      "dependencies": {}
```
Comment (patriksvensson) [inline src/Cake.Testing.Xunit/project.json]: This can be removed.

File: src/Cake.Testing.Xunit/project.json [hunk 2]
```diff
@@ -30,14 +30,10 @@
...
     },
     "net45": {
-      "dependencies": {
-        "xunit.core": "2.2.0-beta2-build3300"
-      },
+      "dependencies": {},
```
Comment (patriksvensson) [inline src/Cake.Testing.Xunit/project.json]: This can be removed.

**Comment by gep13** [comment]
Relates to #1483

**Comment by gep13** [comment]
@phillipsj thanks for making this change!  Will I merge in once all the CI's are finished up.

**Pull request merged**